spec/unit/resource/windows_share_spec.rb

```diff
@@ -42,5 +42,9 @@
     expect(resource.path).to eql("C:\\chef")
     resource.path("C:\\chef")
     expect(resource.path).to eql("C:\\chef")
+    resource.path("C:/chef".freeze)
```

Comment:
I'm not sure if we need these?

Reply:
Okk. will update it. Thanks @tas50

Reply:
If you're trying to make sure the path was frozen then you can just freeze the first one in this spec right?

Reply:
Ok, I'll update it, thanks @tas50

Reply:
Hello @tas50, All tests other than kitchen are passed.